pom.xml

```diff
@@ -58,7 +58,7 @@
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
         <artifactId>bom-2.375.x</artifactId>
-        <version>2133.v2e6c00fe4d61</version>
+        <version>2143.ve4c3c9ec790a</version>
```

Comment:
Not good enough, because the relevant fix is pinned in that line. Need to switch to 2.387.x.